src/components/UserDetailSections/ReadingRoomAccess/ReadingRoomAccess.test.js

```diff
@@ -84,13 +87,40 @@ describe('ReadingRoomAccess', () => {
   it('should filter MCL records "Name" column, based on the string entered in search box', async () => {
     render(<ReadingRoomAccess {...alteredProps} />);
     const inputEl = screen.getByPlaceholderText('ui-users.readingRoom.filter');
-    fireEvent.change(inputEl, { target: { value: 'room1' } });
+    fireEvent.change(inputEl, { target: { value: 'room 1' } });
```

Comment:
[The first sentence in the documentation for `fireEvent` is an explanation of why you should not use fireEvent](https://testing-library.com/docs/dom-testing-library/api-events). You don't _have_ to change this, but please consider the alternative they recommend, [user-event](https://testing-library.com/docs/user-event/intro) for future work.